services/brig/test/integration/Main.hs

```diff
@@ -84,6 +85,7 @@ instance FromJSON BackendConf where
     BackendConf
       <$> o .: "brig"
       <*> o .: "galley"
+      <*> o .: "cargohold"
       <*> o .: "federatorInternal"
       <*> o .: "federatorExternal"
```

Comment:
With so many field of the same type in `BackendConf`, it's easy to confuse one for another. How about introducing a `newtype` wrapper for each one?

Reply:
I'd rather refactor brig's test to look more like galley's and cargohold's (i.e. with a reader monad). But in any case, not in this PR.